api/helpers.go

```diff
@@ -25,3 +25,11 @@ func FeesToBtFeeQuote(minMiningFee float64) *bt.FeeQuote {
 
 	return btFeeQuote
 }
+
+func StringToPointer(input string) *string {
```

Comment:
Have never seen those. Do we really need a function for that?

Reply:
Unfortunately it is not possible to get a pointer to a string like this `&"some string"`
I was looking for a standard library function for that. But didn't find one.

Reply:
Ok, I think I may have found it though: https://pkg.go.dev/go.octolab.org/pointer#ToString

Reply:
`somestring := string("shota")
	ptr := &somestring` or you wanted in one line?